### DISCUSSION TITLE
Scythe Anticheat v2.21.0

### DISCUSSION THREAD

**Discussion by MrDiamond64** [Scythe-Anticheat/Scythe-Anticheat#129] [Announcements]
Scythe Anticheat is now at version v2.21.0!

Supported Minecraft Versions: 1.20.70, 1.20.71

# Changelog
- Added support for MC version 1.20.70
- `trial_spawner` has been added to semi_illegal_items list
- Fix NamespoofB regex not properly saving
- Worldborder now teleports the player instead of applying knockback
- Fixed InvalidSprintA not working
- SpammerA now teleports players to their last known good position
- Scythe alerts can now be logged into the BDS console
- KillauraA now uses a Scripting API implementation
- ScaffoldA now properly works under y level 0
- Added ScaffoldE
- Fix false flags with NoslowA, AutotoolA, SpammerA and ScaffoldC
- Removed InventoryMods/A due to false positives
- Special conditions in player.json now uses the `queue_command` event component instead of hacky solutions utilizing `variant` and `mark_variant`
- Removed Override Command Blocks Enabled
- Bug fixes
- Performance optimizations

By downloading this addon, you agree to our [license](https://github.com/Scythe-Anticheat/Scythe-AntiCheat/blob/main/LICENSE)
Questions? Join our [discord](https://discord.gg/pKyu95mYCT)!

<hr /><em>This discussion was created from the release <a href='https://github.com/Scythe-Anticheat/Scythe-AntiCheat/releases/tag/v2.21.0'>Scythe Anticheat v2.21.0</a>.</em>

**Comment by WaddleDeveloper7** [comment]
You might want to fix namespoof, when I added it to my realm it thought my username was inaccurate and thus I couldn’t join. I had to have one of my friends change the settings for me. My username is “DeweyDuck2010” and it works for Minecraft. Maybe you could fix this? Otherwise great anticheat. Although would recommend fixing up reach because I keep getting false flagged when fighting mobs. Maybe because I have knockback 2 on my sword? I’ll have to test it. Thanks!
